### DISCUSSION TITLE
Service anzeigen lassen

### DISCUSSION THREAD

**Discussion by vwtuner** [DurgNomis-drol/ha_toyota#220] [Q&A]
![ Hochladen von IMG_5427.png...]()

ist es möglich sich das Datum des letzten Service anzeigen zu lassen? Wenn ja wie?
In der my toyota App wird es angezeigt

**Comment by CM000n** [comment]
We have not yet found the appropriate endpoint to query the corresponding data in the `mytoyota` python lib.
If you like, you are welcome to create a feature request 😊 https://github.com/DurgNomis-drol/mytoyota/issues

**Reply by CM000n** [reply]
Ok, I have just found the end point, and my last inspections are also displayed. 😉 
Now we just need to integrate it into the `mytoyota` lib before I we use it for the home assistant custom component.
